libbeat/publisher/client.go

```diff
@@ -76,6 +77,7 @@ func newClient(pub *BeatPublisher) *client {
 		beatMeta: common.MapStr{
 			"name":     pub.name,
 			"hostname": pub.hostname,
+			"version":  version.GetDefaultVersion(),
```

Comment:
@tsg good point, either I go the approach suggested by @andrewkroh or introduce a SetVersion to set the final version here: https://github.com/elastic/beats/pull/2700/files#diff-4cc56dbce8e84b1bc2a025380117788eR139

Looking into it. This should be definitively in sync with the beat version itself.